package.json

```diff
@@ -41,20 +56,13 @@
     "rollup-plugin-serve": "^2.0.2",
     "ts-jest": "^29.1.1",
     "ts-node": "^10.9.1",
+    "tsup": "^7.1.0",
     "typescript": "^5.1.6"
   },
   "files": [
     "build"
   ],
   "engines": {
     "node": ">=16"
-  },
-  "type": "module",
-  "exports": {
-    ".": {
-      "require": "./build/cjs/jwt-decode.js",
-      "import": "./build/esm/jwt-decode.js",
-      "types": "./build/typings/index.d.ts"
-    }
```

Comment:
I've moved this up next to the `main` and `module` definitions, so that all the paths that export stuff are in the same place.